rpc/src/cli.rs

```diff
@@ -13,6 +13,9 @@ pub(crate) enum Commands {
         /// The RPC URL
         #[arg(short = 'u', long, value_hint = ValueHint::Url)]
         rpc_url: String,
+        /// The RPC Tracer Type
+        #[arg(short = 't', long, default_value = "jerigon")]
```

Comment:
nit: you could define an enum to avoid panicing on unexpected string:
```
#[derive(ValueEnum)]
pub enum RpcType {
    Jerigon, 
    Native
}

 #[arg(short = 't', long, default_value_t = RpcType::Jerigon)]
 rpc_type: RpcType,

```